pelops/features/resnet50.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+from keras.applications.resnet50 import ResNet50
+from keras.preprocessing import image
+from keras.applications.resnet50 import preprocess_input
+from keras.models import Model
+
+from PIL import Image as PIL_Image
+from pelops.features.feature_producer import FeatureProducer
+
+# Use global so we only load the resnet model once
+# TODO: find a better way to do this
+resnet_model = None
+
+
+class ResNet50FeatureProducer(FeatureProducer):
+    def __init__(self, chip_producer):
+        global resnet_model
+        super().__init__(chip_producer)
+
+        if not resnet_model:
+            # include_top needs to be True for this to work
+            base_model = ResNet50(weights='imagenet', include_top=True)
+            resnet_model = Model(input=base_model.input,
+                      output=base_model.get_layer('flatten_1').output)
+
+        self.resnet_model = resnet_model
+
+    @staticmethod
+    def preprocess_image(img, x_dim=224, y_dim=224):
```

Comment:
should probably be in an external helper file, with x_dim, y_dim stored inside the class